src/controllers/user.ts

```diff
@@ -83,7 +88,12 @@ const userController = {
         if (!errors.isEmpty()) {
             res.status(statusCodes.MISSING_PARAMS).json(errors.formatWith(errorMessage).array()[0]);
         } else {
-            res.status(statusCodes.SUCCESS).send({msg: "Not implemented"});
+            try{
+                const user = await userDBInteractions.delete(req.params.userId);
+                res.status(statusCodes.SUCCESS).send(user);
```

Comment:
Must check if the user exists before calling delete. If user doesn't exist, return a 404 "User not found"